src/material/core/animation/animation.ts

```diff
@@ -6,7 +6,16 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
-import {ANIMATION_MODULE_TYPE, inject} from '@angular/core';
+import {ANIMATION_MODULE_TYPE, inject, InjectionToken} from '@angular/core';
+
+/** Object used to configure the animation in Angular Material. */
+export interface AnimationsConfig {
+  /** Whether all animations should be disabled. */
+  animationsDisabled: boolean;
+}
+
+/** Injection token used to configure the animations in Angular Material. */
+export const MATERIAL_ANIMATIONS = new InjectionToken<AnimationsConfig>('MATERIAL_ANIMATIONS');
```

Comment:
Technically this could be a `boolean`, but I went with the object literal so we can add more options later if necessary.